app/assets/stylesheets/email.css.scss

```diff
@@ -47,6 +52,10 @@ h4 {
   margin-bottom: 16px;
 }
 
+.mt-40 {
+  margin-top: 40px;
+}
```

Comment:
Since the design system utility styles are now available in the email templates, I think we should use `margin-top-5` instead, which should have the same effect.

(**Edit:** That's assuming `1rem` = `16px`, which is true for most of the IdP, though not 100% about emails, since I believe they're shown with a smaller font size)